rts/lib/libcpuid/libcpuid/cpuid_main.c

```diff
@@ -290,7 +290,12 @@ static bool set_cpu_affinity(logical_cpu_t logical_cpu)
 
 static int get_total_cpus(void)
 {
+#ifdef __OpenBSD__
+	// HW_NCPUONLINE accounts for hyperthreading off/on
```

Comment:
```suggestion
	// HW_NCPUONLINE accounts for hyperthreading off/on,
	// not available on regular linux
```

Reply:
This isn't related to linux. It looks like linux uses sysconf as I linked above, with different header definitions. This code section is inside:
```
#if defined __FreeBSD__ || defined __OpenBSD__ || defined __NetBSD__ || defined __bsdi__ || defined __QNX__
```
In my opinion the HW_NCPUONLINE is clearly enough expressed as OpenBSD-only by the `#ifdef __OpenBSD__` right before it, but if you feel strongly that this needs stating, then it should be something like:
```
// not available on FreeBSD/NetBSD/bsdi/QNX
```
or
```
// only available on OpenBSD
```

Reply:
Alright, in that case it's fine as-is IMO.